core/server/data/importer/importers/data/settings.js

```diff
@@ -90,6 +102,48 @@ class SettingsImporter extends BaseImporter {
             return data;
         });
 
+        for (const key in deprecatedSupportedSettingsOneToManyMap) {
+            const deprecatedSetting = this.dataToImport.find(setting => setting.key === key);
+
+            // NOTE: filtering before mapping in case the key name is the same
+            this.dataToImport = _.filter(this.dataToImport, data => (key !== data.key));
+
+            if (deprecatedSetting) {
+                let deprecatedSettingValue;
+
+                try {
+                    deprecatedSettingValue = JSON.parse(deprecatedSetting.value);
+                } catch (err) {
+                    this.problems.push({
+                        message: `Failed to parse the value of ${deprecatedSetting.key} setting value`,
+                        help: this.modelName,
+                        context: deprecatedSetting
+                    });
+                }
+
+                if (deprecatedSettingValue) {
+                    deprecatedSupportedSettingsOneToManyMap[key].forEach(({from, to}) => {
+                        let value = _.isObject(deprecatedSettingValue)
+                            ? _.get(deprecatedSettingValue, from)
+                            : deprecatedSetting.value;
+
+                        this.dataToImport.push({
+                            id: deprecatedSetting.id || ObjectId.generate(),
```

Comment:
This could cause duplicate `id`'s if the deprecated setting is split into multiple settings

Reply:
Hm.. :thinking:  Yup, very good point!